resources/dashboards/Capture-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "title": "Capture Dashboard",
```

Comment:
Alex thinks the Analyze Table on this dashboard is messed up. Need to look at a 2.8.2 box to see what the actual query is